### DISCUSSION TITLE
which one is better between  ethers.js and web3.js

### DISCUSSION THREAD

**Discussion by bstar0406** [bstar0406/bstar-web3modal-nft#10] [General]
I have a question.
As you can see, many developer use web3.js and ethers.js for Dapp development but I don't know well, what I have to choose.

**Comment by uStar10** [comment]
In my opinion, web3.js is the better choice because, we can find many sample of web3.js but it is difficult to find ethers.js ones
So I recommend web3.js

**Reply by bstar0406** [reply]
how about the performance?

**Comment by bstar0406** [comment]
I found the result!
While web3. js provides a single instantiated web3 object with methods for interacting with the blockchain, ethers. js separates the API into two separate roles. The provider , which is an anonymous connection to the ethereum network, and the signer , which can access the private key and sign the transactions.
ethers.js is better choice for the future
